zokrates_book/src/concepts/imports.md

```diff
@@ -6,22 +6,22 @@ You can separate your code into multiple ZoKrates files using `import` statement
 
```

Comment:
The changes here are inconsistent with the examples in the pr, where the `.zok` ending is no longer required in import statements.